plugins/E1.31/e131controller.cpp

```diff
@@ -25,18 +25,18 @@
 #define TRANSMIT_FULL    "Full"
 #define TRANSMIT_PARTIAL "Partial"
 
-E131Controller::E131Controller(QNetworkInterface const& interface, QString const& ipaddr,
-                               Type type, quint32 line, QObject *parent)
+E131Controller::E131Controller(QNetworkInterface interface, QNetworkAddressEntry address,
```

Comment:
Don't you want to keep the "const & interface" to avoid temporary object creation? The same for address?